internal/vendor/wiz.go

```diff
@@ -141,10 +143,28 @@ var Environment = []string{
 type ProjectCloudOrganizationLinkInput struct {
 	CloudOrganization string         `json:"cloudOrganization"`
 	Environment       string         `json:"environment"` // enum Environment
+	ResourceGroups    []string       `json:"resourceGroups"`
 	ResourceTags      []*ResourceTag `json:"resourceTags"`
 	Shared            bool           `json:"shared"`
 }
 
+// ProjectCloudAccountLinkInput struct
+type ProjectCloudAccountLinkInput struct {
+	ID             string         `json:"cloudAccount"`
+	Environment    string         `json:"environment"` // enum Environment
+	ResourceGroups []string       `json:"resourceGroups"`
```

Comment:
The GraphQL schema allows null values.

```suggestion
	ResourceGroups []string       `json:"resourceGroups,omitempty"`
```